### ISSUE TITLE
demo deployment isnt live?

### ISSUE THREAD

**Issue by Mr-DJ** [ArpitPy/whistleblower#24]
![image](https://user-images.githubusercontent.com/28898632/197858924-d3713204-ab38-466d-bac1-59ca9f6ac8c5.png)

**Comment by ArpitPy** [comment]
Yup recently made a very aggressive scan on all the websites hosted on it and for some reason several project websites got flagged as phishing website so it's down. I will soon host it on vercel on netlify.
